README.md

```diff
@@ -132,12 +132,9 @@ More detailed documentation is available here: https://oneapi-src.github.io/unif
 
 ### Memory providers
 
-#### Coarse Provider
+#### Fixed memory Provider
```

Comment:
Fixed Memory Provider, like in docs

Reply:
👍

Reply:
This is exactly how other providers are named in this section - if it's not correct we should fix it in separate PR

Reply:
`Fixed memory provider` is OK - just like others